packages/ibm-products/src/components/NonLinearReading/NonLinearReading.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useRef, useState } from 'react';
+
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { ChevronDown } from '@carbon/react/icons';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import uuidv4 from '../../global/js/utils/uuidv4';
+import { pkg } from '../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--non-linear-reading`;
+const componentName = 'NonLinearReading';
+
+// Default values for props
+const defaults = {
+  theme: 'light',
+};
+
+/**
+ * Use non-linear reading when space is limited to share a
+ * brief, at-a-glance, summary of a concept that may require
+ * more explanation for some users.
+ */
+export let NonLinearReading = React.forwardRef(
+  (
+    { children, className, definition, theme = defaults.theme, ...rest },
+    ref
+  ) => {
+    const [isOpen, setOpen] = useState(false);
+    const contentId = useRef(uuidv4()).current;
+
+    const handleToggle = () => {
+      setOpen((prevState) => !prevState);
+    };
+
+    return (
+      <span
+        {...rest}
+        className={cx(blockClass, `${blockClass}__${theme}`, className)}
+        ref={ref}
+        role="main"
```

Comment:
I don't think this should have a role of `main` since this likely isn't the primary content of the page.

Reply:
I'm new to `roles` and I'm open to options.

Reply:
[Here's the list](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles). Not sure what feels most appropriate, `definition` possibly?

Reply:
I couldn't find anything that matched perfectly, either.

Unfortunately, this UX doesn't fit any existing, standard component behaviour.

It's not explicit, but it doesn't seem like the definition should be hidden from the user. [digitala11y.com](https://www.digitala11y.com/definition-role/) explains "the definition of a term that is described in the _natural flow of the narrative_ is associated with the term itself using the aria-labelledby attribute".

I was thinking that this component behaved a lot like an inline version of Carbon's Accordion component ([GitHub](https://github.com/carbon-design-system/carbon/tree/main/packages/react/src/components/Accordion)), but it and its subcomponents don't have roles at all, and they're just buttons, divs, and paragraphs. Zero semantic HTML to tie them together, just `aria-controls` associated to `ids`.

I don't see a clear path here.

Reply:
In that case, I think for now just omit the role until we can find something that makes more sense. Don't want this to block you.